### DISCUSSION TITLE
Deploy Saleor Dashboard to Vercel

### DISCUSSION THREAD

**Discussion by BrunoVanDamme** [saleor/saleor#7947] [Deployment] [answered]
Hi... I've recently discovered Saleor and got the "saleor-platform" (core, storefront and dashboard) running locally.
I'm having troubles with deploying the dashboard on Vercel. I've forked the repo "saleor-dashboard" and imported it in Vercel. I've set the API_URI environment variable in the Vercel platform. I'm getting an error when building... something with public folder not set. I've looked in the "saleor-storefront" repo and saw there is a public folder integrated starting from release 3.0. I couldn't find that folder in the "saleor-dashboard" repo. Any advice to get the dashboard running on Vercel? Is it as simple as adding an empty "public" folder are do i need to do other things? Thanks!

**Comment by daltonsena** [comment, marked as answer]
Hi @BrunoVanDamme!

In the settings area, you need to configure your "output directory" and the "Framework preset" in the "Build & Development Settings" section. (https://vercel.com/[workspace]/[project]/settings)

I only made these deployment settings for tests purpose, but for me, this setting works like a charm to saleor-dashboard running on the Vercel platform:

![Screen Shot 2021-08-26 at 19 20 25](https://user-images.githubusercontent.com/247386/131044438-eba02846-10e6-4601-8b05-ff05ebd33e7f.png)

**Reply by BrunoVanDamme** [reply]
Hi @daltonsena , 

Your solution worked perfect! Thank you very much!

**Reply by nickradford** [reply]
This gets the dashboard running, but I've been having issues when you either reload a page, or need to open a url that's not the root path. 

For example trying to go to http://mySaleorDashboard.app/configuration results in this lovely 404 from Vercel

![image](https://user-images.githubusercontent.com/222221/138197389-512ba95d-58eb-4c7f-b013-aa8b79513e90.png)

**Reply by nickradford** [reply]
Solution:

Add a `vercel.json` to the root of the saleor-dashboard repository

```json
{
  "rewrites": [{ "source": "/:path*", "destination": "/index.html" }]
}
```
